src/django/api/templates/mail/facility_claim_profile_update_contributor_notice_body.txt

```diff
@@ -0,0 +1,12 @@
+Hi,
+
+You're receiving this email because a facility on one of your facility lists
+has had its facility claim data updated.
+
+The facility is:
+
+  - Facility: {{ facility_name }}, {{ facility_address }}
```

Comment:
Consider including the full facility country name in these notifications.

Reply:
Sounds good. I added this in 0d3d6a29a1fa3bbfdafd95b0d042eaa5e1c58b2a and also added the full country name to the previous set of facility claim emails.